DataFormats/Detectors/TRD/include/DataFormatsTRD/HelperMethods.h

```diff
@@ -54,6 +54,13 @@ struct HelperMethods {
     printf("%02i_%i_%i\n", det / constants::NCHAMBERPERSEC, (det % constants::NCHAMBERPERSEC) / constants::NLAYER, det % constants::NLAYER);
   }
 
+  static void printSectorStackLayerSide(int hcid)
+  {
+    // for a given half-chamber number prints SECTOR_STACK_LAYER_side
+    int det = hcid / 2;
+    printf("%02i_%i_%i side: %i\n", det / constants::NCHAMBERPERSEC, (det % constants::NCHAMBERPERSEC) / constants::NLAYER, det % constants::NLAYER, hcid % 2);
+  }
+
```

Comment:
I have been wondering if we should have some general formatting goodies, either for fmt::format or C++ ostreams.
I think we have some ostream operators for digits (and tracklets?). 
I would also suggest to write this as e.g. '01_2_3A'. or '04_5_1B'.

Reply:
Ok